packages/waku/src/cli.ts

```diff
@@ -149,7 +152,7 @@ async function runStart() {
     return c.text('404 Not Found', 404);
   });
   const port = parseInt(values.port || '8080', 10);
-  await startServer(app, port);
+  await startServer(honoEnhancer(app), port);
```

Comment:
Do we need it for `runStart`?

Reply:
It could be useful for `runStart`. Maybe the `cmd` and `config` should be additional arguments to `honoEnhancer` too.

Reply:
`config` feels somewhat recursive. `cmd` seems good, but I actually removed it from `middleware` before, so let's leave it until we see real issues.

Reply:
There's another concern: loadConfig will be defined in dist/entries.js, so it's dual modules, which may cause something unexpected.

Reply:
OK. No problem. This is not an issue for any of my current use cases.